src/api-reference/common/event-pubsub/travel-search-event.markdown

```diff
@@ -69,16 +71,18 @@ Name|Type|Format|Description
 
 Name|Type|Format|Description
 ---|---|---|---
-`cityName`|`String`|YYYY-MM-DD|Name of the city .
-`cityIATACode`|`String`|YYYY-MM-DD|IATA code of the city.
+`cityName`|`String`|Frankfurt|Name of the city .
```

Comment:
`Frankfurt` and `FRA` aren't formats, they are examples. Edit this as follows and you will be good:

```
`cityName`|`String`|-|Name of the city. Example: Frankfurt
`cityIATACode`|`String`|-|IATA code of the city. Example: FRA
```

Reply:
@charlieconcurI have changed the document. please, do have a look. if you have more changes you want please let me know.